### PR TITLE
[WIP] [FEATURE] re-enable indented view of localized record variants

### PR DESCRIPTION
**Pull request by jpmschuler**
As the complete logic for localised records is already there and they are just not shown, I propose to show the localised records by adapting the `Node.html`. The edit areas and buttons work, so this allows the pre-v8 status quo. This leaves two major points open:

1. adding translations as AJAX action
2. different language views (e.g. the option focus on translation X). I know that TV and TVP offered them for a long time, but I neither used the other options nor did we mention them in customer manuals at all, because this indented view is perfectly fine as long as there are only like 2-3 languages in the system.

Additionally the unused elements are now filtered to remove node translations, instead of also mentioning them. This not only removes translations of used elements (which need to go away) but also those of unused - but refactoring it to show the latter in an l10n-indented way would be more work with just a few benefit, considering this easy win.

### CODE REVIEW

**Comment by opi99** [comment]
Which form of translation do you use?

**Comment by jpmschuler** [comment]
Connected mode. However I didn't do anything to the data, I just added the already existing data to the fluid output, so this should adhere to the defaults of v7 and still allow a change of what should be displayed.

Bildschirmfoto 2022-07-23 um 01 39 04
Bildschirmfoto 2022-07-22 um 23 46 44

**Comment by opi99** [comment]
Hmmm, ok, that might work

**Comment by opi99** [comment]
But for the moment, I'd like to have this in 8.2.0 and not 8.1.2. Would that be fine?

**Comment by jpmschuler** [comment]
A delay is fine, this PR allows to `composer-patches` it in urgent cases. 

Additionally it seems there is still a hiccup somewhere. I have multiple TVP-templates in the same instance with each multiple sites and have one template where the translations are missing and a click on "create" gives the valid error "already existing". Will check for differences.

**Comment by jpmschuler** [comment]
Updated PR to only show localisations if relevant (e.g. tt_content "localise me" buttons won't be shown if the page translation for that language doesn't exist yet, hidden is ok).

**Comment by jpmschuler** [comment]
Streamlined design, allowing localization editing for pages using the same UX known from content
Extracted page as special node type to prevent double page titles
example

Marking PR as WIP until new features thoroughly tested in all LTS and until sidebar has button to have old non-l10n  behaviour ;-)

**Comment by jpmschuler** [comment]
example2
After a lot of testing around I decided to drop ajax based page record editing as this can have disastrous implications, instead a real page reload is more suitable.

File: Classes/Utility/TemplaVoilaUtility.php [hunk 1]
```diff
@@ -186,6 +193,137 @@ public static function getAvailableLanguages($id = 0, $setDefault = true, $setMu
         return $languages;
     }
 
+<<<<<<< HEAD
+=======
+    /**
```
Comment (opi99) [inline Classes/Utility/TemplaVoilaUtility.php:198]: There seams a rebase error

**Review by opi99 — COMMENTED** [review]
There is a rebase error

**Pull request merged**